### PR TITLE
17708 - Displaying labels beside statements, get total due amount and date

### PR DESCRIPTION
**Pull request by rodrigo-barraza**
*Issue #:17708*
https://github.com/bcgov/entity/issues/17708

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/account-settings/statement/Statements.vue [hunk 1]
```diff
@@ -338,17 +365,25 @@ export default defineComponent({
 </script>
 
 <style lang="scss" scoped>
+.label {
+  background: #1669bb;
```
Comment (seeker25) [inline auth-web/src/components/auth/account-settings/statement/Statements.vue:369]: use `$app-blue`

File: auth-web/src/stores/org.ts [hunk 2]
```diff
@@ -699,6 +699,13 @@ export const useOrgStore = defineStore('org', () => {
     return result
   }
 
+  async function getStatementsSummary () {
+    const response = await PaymentService.getStatementsSummary(state.currentOrganization.id)
```
Comment (seeker25) [inline auth-web/src/stores/org.ts:703]: Probably should have a try catch, that way the rest of the page keeps working even if the call fails. Here's an example where call fails and causes other code to have issues:

https://github.com/bcgov/sbc-auth/pull/2577

File: auth-web/src/components/auth/account-settings/statement/Statements.vue [hunk 3]
```diff
@@ -338,17 +365,25 @@ export default defineComponent({
...
+  color: white;
+  border-radius: 4px;
+  &.overdue {
+    background: #D3272C;
```
Comment (seeker25) [inline auth-web/src/components/auth/account-settings/statement/Statements.vue:373]: Might be worth defining this in the SCSS, I see it's used in a few spots now

File: auth-web/src/components/auth/account-settings/statement/Statements.vue [hunk 4]
```diff
@@ -178,6 +186,15 @@ export default defineComponent({
       }
     ])
 
+    const isStatementNew = (item) => {
+      return Math.max(...statementsList.value.map(statement => statement.id)) === item.id
+    }
+
+    const isStatementOverdue = (item) => {
+      return (paymentOwingAmount.value && paymentDueDate.value) &&
```
Comment (seeker25) [inline auth-web/src/components/auth/account-settings/statement/Statements.vue:194]: Would this show them all as overdue? - do we need to change the summary endpoint to show which are overdue?

**Comment by seeker25** [comment]
package json + package lock

File: auth-web/src/assets/scss/theme.scss [hunk 5]
```diff
@@ -79,6 +79,7 @@ $app-lt-blue: #E2E8EE; // LOW OPACITY PRIMARY BLUE
 $app-background-blue: #E4EDF7; // Hover Background Blue
 $app-dk-blue: #38598a;
 $app-green: #2E8540;
+$app-red: #D3272C;
```
Comment (seeker25) [inline auth-web/src/assets/scss/theme.scss:82]: excellent

File: auth-web/src/components/auth/account-settings/statement/Statements.vue (hunk 4 above)
Comment (seeker25) [reply auth-web/src/components/auth/account-settings/statement/Statements.vue:194]: sounds good I'll fix this on the BE

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2586-6r7z1scz.web.app

File: auth-web/src/components/auth/account-settings/statement/Statements.vue [hunk 6]
```diff
@@ -140,13 +148,13 @@ export default defineComponent({
     }
   },
   setup (props, { root }) {
+    const ITEMS_PER_PAGE = 5
+    const PAGINATION_COUNTER_STEP = 4
     const orgStore = useOrgStore()
+    const currentOrganization: Organization = orgStore.currentOrganization
```
Comment (seeker25) [inline auth-web/src/components/auth/account-settings/statement/Statements.vue:154]: What happens if someone changes their account while on the statements page?

**Review by seeker25 — APPROVED** [review]
Before merging: What happens if someone switches their account while on the statements page?

**Pull request merged**